expression/expression.go

```diff
@@ -243,17 +243,42 @@ func VecEvalBool(ctx sessionctx.Context, exprList CNFExprs, input *chunk.Chunk,
 			return nil, nil, err
 		}
 
+		j := 0
+		if eType == types.ETInt {
+			if err = expr.VecEvalInt(ctx, input, buf); err != nil {
+				return nil, nil, err
+			}
+
+			i64s := buf.Int64s()
+			for i := range sel {
+				if buf.IsNull(i) {
+					nulls[sel[i]] = true
+					sel[j] = sel[i]
+					j++
+					continue
+				}
+				if i64s[i] == 0 {
+					continue
+				}
+				sel[j] = sel[i] // this row passes this filter
+				j++
+			}
+			sel = sel[:j]
+			input.SetSel(sel)
+			globalColumnAllocator.put(buf)
+			continue
+
```

Comment:
this empty line is meaningless, can we remove it?

Reply:
Update. PTAL